client/components/CreateEditAgendaForm.jsx

```diff
@@ -3,15 +3,18 @@ import { fields, Toggle, AuditInformation } from '../components'
 import { connect } from 'react-redux'
 import { Field, reduxForm } from 'redux-form'
 import PropTypes from 'prop-types'
-import { get } from 'lodash'
+import { get, isUndefined } from 'lodash'
 import * as actions from '../actions'
 import { ChainValidators, RequiredFieldsValidatorFactory, MaxLengthValidatorFactory } from '../validators'
 
 export class Component extends React.Component {
 
     constructor(props) {
         super(props)
-        this.state = { isEnabled: this.props.initialValues.is_enabled }
+        this.state = {
+            isEnabled: isUndefined(this.props.initialValues.is_enabled)
```

Comment:
in this case I use `get(this.props.initialValues, 'is_enabled', true)`
but it's matter of taste :)